content/blog/kubernetes-monitoring-using-prometheus-and-grafana-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,419 @@
+* Both Prometheus and Grafana services are deployed as *NodePort* service types. Those services will be mapped to the gateway host with automatically generated ports for easy access and service configuration.
+
+#### Initialize working directory
+
+With above *main.tf* config file, the working directory can be initialized by running the following command: 
+
+```shell
+    $ terraform init
+    
+    Initializing the backend...
+    
+    Initializing provider plugins...
+    - Finding hpe/hpegl versions matching ">= 0.2.2"...
+    - Finding latest version of hashicorp/helm...
+    - Installing hpe/hpegl v0.3.17...
+    - Installed hpe/hpegl v0.3.17 (signed by a HashiCorp partner, key ID D1F277A1AC66CE3D)
+    - Installing hashicorp/helm v2.10.1...
+    - Installed hashicorp/helm v2.10.1 (signed by HashiCorp)
+    
+    Partner and community providers are signed by their developers.
+    If you'd like to know more about provider signing, you can read about it here:
+    https://www.terraform.io/docs/cli/plugins/signing.html
+    
+    Terraform has created a lock file .terraform.lock.hcl to record the provider
+    selections it made above. Include this file in your version control repository
+    so that Terraform can guarantee to make the same selections by default when
+    you run "terraform init" in the future.
+    
+    Terraform has been successfully initialized!
+    
+    You may now begin working with Terraform. Try running "terraform plan" to see
+    any changes that are required for your infrastructure. All Terraform commands
+    should now work.
+    
+    If you ever set or change modules or backend configuration for Terraform,
+    rerun this command to reinitialize your working directory. If you forget, other
+    commands will detect it and remind you to do so if necessary.
+```
+
+F﻿rom the ouput of the command run, in addition to the HPE GreenLake Terraform provider *hpegl*, the provider *helm* is also installed to the Terraform working directory.
+
+#### Deploy Prometheus and Grafana
+
+Typing the following command to apply the Terraform configuration and deploy Prometheus and Grafana to the K8s cluster while responding *yes* at the prompt to confirm the operation. You may try first a dry run, by running *terraform plan*, to preview the changes to your infrastructure based on the data you provide in your Terraform file. 
```

Comment:
Change "Typing" to "Type"

For the second sentence, rephrase to:
"You may want to first try a dry run to preview the changes to your infrastructure based on the data you provide in your Terraform file by running terraform plan."  (a few words have been added/changed and the structure of the sentence has been modified)